examples/next-prisma-starter-websockets/src/server/wssDevServer.ts

```diff
@@ -1,11 +1,10 @@
 import { createContext } from './context';
 import { appRouter } from './routers/_app';
 import { applyWSSHandler } from '@trpc/server/adapters/ws';
-import fetch from 'node-fetch';
 import ws from 'ws';
 
 if (!global.fetch) {
-  (global as any).fetch = fetch;
+  (global as any).fetch = require('node-fetch');
```

Comment:
This doesn't make sense. You can't mix `import` and `require` (following the spec, but I guess webpack/Next allows it). I suggest keeping the `node-fetch` import and adding a comment saying to remove it instead.

Reply:
> Webpack allows it

That's the point. Right now, the example runs on all Node versions, while importing `node-fetch` fails on v18. The alternative is to use a dynamic `import()`, but without `topLevelAwait` (via webpack, once again :P) the whole server would need to be wrapped in an async IIFE of sorts.